### ISSUE TITLE
Only first value is communicated to screen readers for read-only inputs

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3098]
In Profile Editor
Within the read-only inputs for "Properties", "Assets" and "Keybindings", Only the first value is communicated to screen reader users while the remaining values cannot be read at all.

**Suggested Solution:** Use list views in place of multi-line text fields to display available options. Potentially, accompany each list view with controls to edit the existing settings, such as "Add" and "Remove" buttons, as discussed in the last observation in this section.

Severity level 1 from PAC
